packages/sx.js/src/clients/offchain/types.ts

```diff
@@ -41,6 +41,7 @@ export type Vote = {
   authenticator: string;
   strategies: StrategyConfig[];
   proposal: string;
-  choice: Choice;
+  choice: Choice | number | number[];
```

Comment:
I think I'd just extend `Choice` type to include those extra types and and replace `Choice | number | number[]` back to `Choice` across the app.